src/components/services/controllers.mocked.test.tsx

```diff
@@ -0,0 +1,198 @@
+  });
+
+
+  it('should throw an error if service is not supporting logs', async () => {
+    CFClient.prototype.userServices.mockReturnValueOnce(Promise.resolve([ mockCustomService ]));
+    CFClient.prototype.space.mockReturnValueOnce(Promise.resolve(mockSpace));
+    CFClient.prototype.organization.mockReturnValueOnce(Promise.resolve(mockOrganization));
+    CFClient.prototype.serviceInstance.mockReturnValueOnce(Promise.resolve(mockServiceInstance));
+    CFClient.prototype.service.mockReturnValueOnce(Promise.resolve(mockServiceRedis));
+
+    await expect(
+      listServiceLogs(ctx, {
+        organizationGUID: 'ORG_GUID', serviceGUID: 'SERVICE_INSTANCE_GUID', spaceGUID: 'SPACE_GUID',
+      }),
+    ).rejects.toThrowError(/Service Logs are only available for Postgres and MySQL instances/);
+  });
+
+  it('should successfully list out all the files', async () => {
+    CFClient.prototype.userServices.mockReturnValueOnce(Promise.resolve([ mockCustomService ]));
+    CFClient.prototype.space.mockReturnValueOnce(Promise.resolve(mockSpace));
+    CFClient.prototype.organization.mockReturnValueOnce(Promise.resolve(mockOrganization));
+    CFClient.prototype.serviceInstance.mockReturnValueOnce(Promise.resolve(mockServiceInstance));
+    CFClient.prototype.service.mockReturnValueOnce(Promise.resolve(mockServicePostgres));
+    // @ts-ignore
+    RDS.mockReturnValueOnce({
+      describeDBLogFiles: () => ({
+        promise: async () => await Promise.resolve({
+          DescribeDBLogFiles: [ { LastWritten: 1578837540000, LogFileName: 'file-one', Size: 73728 } ],
+        }),
+      }),
+    });
+
+    const response = await listServiceLogs(ctx, {
+      organizationGUID: 'ORG_GUID', serviceGUID: 'SERVICE_INSTANCE_GUID', spaceGUID: 'SPACE_GUID',
+    });
+
+    expect(response.body).toContain('mydb - Service Metrics Logs');
+    expect(response.body).toContain('file-one');
+  });
+
+  it('should display info about lack of logs', async () => {
+    CFClient.prototype.userServices.mockReturnValueOnce(Promise.resolve([ mockCustomService ]));
+    CFClient.prototype.space.mockReturnValueOnce(Promise.resolve(mockSpace));
+    CFClient.prototype.organization.mockReturnValueOnce(Promise.resolve(mockOrganization));
+    CFClient.prototype.serviceInstance.mockReturnValueOnce(Promise.resolve(mockServiceInstance));
+    CFClient.prototype.service.mockReturnValueOnce(Promise.resolve(mockServicePostgres));
+    // @ts-ignore
+    RDS.mockReturnValueOnce({
+      describeDBLogFiles: () => ({
+        promise: async () => await Promise.resolve({}),
+      }),
+    });
+
+    const response = await listServiceLogs(ctx, {
+      organizationGUID: 'ORG_GUID', serviceGUID: 'SERVICE_INSTANCE_GUID', spaceGUID: 'SPACE_GUID',
+    });
+
+    expect(response.body).toContain('mydb - Service Metrics Logs');
+    expect(response.body).toContain('There are no log files available at this time. Please comeback later.');
```

Comment:
```suggestion
    expect(response.body).toContain('There are no log files available at this time. Please come back later.');
```

Typo, and I'm not sure we need the user to come back later

Reply:
Deleted.